test/kb_Bowtie2_Aligner_test.py

```diff
@@ -195,3 +223,38 @@ def test_bowtie2_aligner(self):
         self.assertIn('report_info', res)
         self.assertIn('report_name', res['report_info'])
         self.assertIn('report_ref', res['report_info'])
+
+
+    def test_bowtie2_aligner_with_readsset(self):
+        assembly_ref = self.loadAssembly()
+        se_lib_ref = self.loadSingleEndReads()
```

Comment:
Couldn't this be done only once in a test setup class?

Reply:
The loading is done only once. Only the ref is returned the subsequent times it is called.